src/releng_channel_status/releng_channel_status/templates/release.html

```diff
@@ -0,0 +1,53 @@
+<div class="row">
+    <div class="col">
+        <div class="card">
+            <div class="card-header">
+                {{ label }} {{ release.name }}
+            </div>
+            <div class="card-body">
+                <table class="table">
+                  <thead>
+                    <tr>
+                      <th>Platforms</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {% for platform in release.platforms %}
+                        <tr>
+                          <td>
+                            
+                            <table class="table">
+                              <thead>
+                                <tr>
+                                    <th colspan="3">{{ platform.name }}</th>
+                                </tr>
+                                <tr>
+                                  <th>Locale</th>
+                                  <th>BuildID</th>
+                                </tr>
+                              </thead>
+                              <tbody>
+                                {% for locale_name, locale in platform.locales.items() %}
```

Comment:
I don't think we need to show BuildID per locale, yet. AFAIK, we have usually have issues related to platforms. I don't remember freezing a locale on Nightly. @mozbhearsum may have more insights, though.

Reply:
The only case I can think of where buildids could be different by locale is when a locale gets dropped. I think it's okay to ignore this case for now.